users/strelaysrv.rst

```diff
@@ -74,12 +74,12 @@ as root or a privileged user, see `Running on port 443 as an unprivileged user`_
 below. In principle something similar to this should work on a Linux/Unix
 system::
 
-    $ sudo useradd relaysrv
-    $ sudo mkdir /etc/relaysrv
-    $ sudo chown relaysrv /etc/relaysrv
-    $ sudo -u relaysrv /usr/local/bin/relaysrv -keys /etc/relaysrv
+    $ sudo useradd strelaysrv
```

Comment:
Should we suggest syncthing-relaysrv as a user here to reflect syncthing-relaysrv.service?